src/modules/dashboard/components/ActionsPage/StakingWidget/StakingWidget.tsx

```diff
@@ -0,0 +1,299 @@
+     */
+    hasRegisteredProfile &&
+    /*
+     * User has enough reputation to stake
+     */
+    bigNumberify(maxUserStake).gt(0) &&
+    bigNumberify(maxUserStake).gte(bigNumberify(minUserStake)) &&
+    /*
+     * Motion can be still staked (ie: amount left to stake)
+     */
+    remainingToStakeSafe > 0 &&
+    /*
+     * Activated tokens are more than the minimum required stake amount
+     */
+    userActivatedTokens >= userStakeBottomLimit &&
+    /*
+     * Has activated tokens
+     */
+    userActivatedTokens > 0;
+
+  return (
+    <div className={styles.main}>
+      <ActionForm
+        initialValues={{
+          amount: userStakeBottomLimit,
+        }}
+        validationSchema={validationSchema}
+        submit={ActionTypes.COLONY_MOTION_STAKE}
+        error={ActionTypes.COLONY_MOTION_STAKE_ERROR}
+        success={ActionTypes.COLONY_MOTION_STAKE_SUCCESS}
+        transform={transform}
+        onSuccess={handleSuccess}
+      >
+        {({ values }) => (
+          <div className={styles.wrapper}>
+            <div className={styles.title}>
+              <Heading
+                text={MSG.title}
+                className={styles.title}
+                appearance={{ size: 'normal', theme: 'dark', margin: 'none' }}
+              />
+              <QuestionMarkTooltip
+                tooltipText={MSG.stakingTooltip}
+                className={styles.help}
+                tooltipClassName={styles.tooltip}
+                tooltipPopperProps={{
+                  placement: 'right',
+                }}
+              />
+            </div>
+            <p className={styles.description}>
+              <FormattedMessage {...MSG.description} />
+            </p>
+            <span className={styles.amount}>{`${parseFloat(
+              values.amount,
+            ).toFixed(2)} ${nativeToken?.symbol}`}</span>
+            <div className={styles.sliderContainer}>
+              <Slider
+                name="amount"
+                value={values.amount}
+                min={userStakeBottomLimit}
+                max={
+                  remainingToStakeSafe < userStakeBottomLimit
+                    ? userStakeBottomLimit
+                    : remainingToStakeSafe
+                }
+                limit={
+                  userStakeTopLimit < userActivatedTokens
+                    ? userStakeTopLimit
+                    : userActivatedTokens
+                }
+                step={0.01}
+                disabled={!canUserStake}
+              />
+            </div>
+            <div className={styles.buttonGroup}>
+              <Button
+                appearance={{ theme: 'primary', size: 'medium' }}
+                type="submit"
+                disabled={!canUserStake}
+                text={MSG.stakeButton}
+              />
+              <Button
+                appearance={{ theme: 'danger', size: 'medium' }}
+                text={MSG.objectButton}
+                disabled={!canUserStake}
```

Comment:
I think that you can't use the same condition for object button as for the stake button because if the yay side is fully staked, we can still object and stake objection. I think it will work if we add a check for the Nay side.
But I can do it in my branch.

Reply:
Sure, you're in control of the Objections part anyway :)